tests/test_cli.py

```diff
@@ -115,57 +114,60 @@ def test_command_list_node(workdir, mocker):
     ('traintuple', ['--objective-key', 'foo', '--algo-key', 'foo', '--dataset-key', 'foo',
                     '--data-samples-path']),
     ('testtuple', ['--traintuple-key', 'foo', '--data-samples-path'])]
-)
+                         )
 def test_command_add(asset_name, params, workdir, mocker):
     method_name = f'add_{asset_name}'
 
-    with tempfile.NamedTemporaryFile(mode='w', suffix='.json') as file,\
-            mock_client_call(mocker, method_name, response={}) as m:
-        json.dump({}, file)
-        file.seek(0)
-
-        client_execute(workdir, ['add', asset_name] + params + [file.name])
+    with mock_client_call(mocker, method_name, response={}) as m:
+        json_file = workdir / "temp_file.json"
+        with open(str(json_file), 'w') as f:
+            json.dump({}, f)
+        client_execute(workdir, ['add', asset_name] + params + [str(json_file)])
     assert m.is_called()
 
-    with tempfile.NamedTemporaryFile(suffix='.md') as file:
-        res = client_execute(workdir, ['add', asset_name] + params + [file.name], exit_code=2)
-        assert re.search(r'File ".*" is not a valid JSON file\.', res)
+    file = workdir / "temp_file.txt"
```

Comment:
From my point of view, we could use a better name for this variable (for instance `invalid_json_file`).
By reading the code, we would directly understand that the first test is done with a valid file and that we repeat it with an invalid file.

Reply:
Yes, I changed that for the next commit 👌